main/systemd.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "strings"
+    "path/filepath"
+)
+
+func contains(distros []string, distro string) bool {
+    if distro != "" {
+        for _, v := range distros {
+            if v == distro {
+                return true
+            }
+        }
+    }
+    return false
+}
+
+func get_distro() string {
+    path, _ := os.Getwd()
+    output, err := exec.Command("sudo", filepath.Join(path, "scripts/distro_script.sh")).Output()
+    if err != nil {
```

Comment:
Is this condition true `err != nil` if the exit code is non-0? If so, what's the error? Does it just contain the non-0 exit code?

Reply:
Yes, it does. I see error codes (1, 123, etc..) logging on failure cases

Reply:
cool thanks